### PR TITLE
docs (encrypt): Add changelog with some history

### PR DESCRIPTION
**Pull request by jimlambrt**

### CODE REVIEW

File: filters/encrypt/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Changelog
+
+## Next 
+### New and Improved
+- Filter map fields which are not tagged. ([PR](https://github.com/hashicorp/go-eventlogger/pull/63))
+- Setting filters to no operation. ([PR](https://github.com/hashicorp/go-eventlogger/pull/61))
+
+##  filters/encrypt/v0.1.3
+### New
```
Comment (louisruch) [inline filters/encrypt/CHANGELOG.md:9]:
```suggestion
### New and Improved
```

**Review by louisruch — COMMENTED** [review]
Did we want to add `(date)` to each release as we do in other projects?

File: filters/encrypt/CHANGELOG.md (hunk 1 above)
Comment (jimlambrt) [reply filters/encrypt/CHANGELOG.md:9]: ty!

**Review by louisruch — APPROVED** [review]
LGTM :shipit:

**Pull request merged**